includes/admin/settings/register-settings.php

```diff
@@ -24,9 +24,14 @@
 function edd_get_option( $key = '', $default = false ) {
 	global $edd_options;
 
-	$value = ! empty( $edd_options[ $key ] )
-		? $edd_options[ $key ]
-		: $default;
+	// Special case for tax_rate
+	if ( 'tax_rate' === $key ) {
```

Comment:
@SimonChaw I'm not sure if I like option-specific code here as it sets a precedent which could result in a lot of options being handled this way in the future. I think we can likely just leave this out entirely as the only place `edd_get_option( 'tax_rate' )` is called is in tax-functions.php, and you've handled it there. If I'm missing another reason this is here, let me know. If you still think this should be here, I would probably prefer it as a filter hooked to `edd_get_option` from tax-functions.php.

Reply:
I agree, this is a good point. Doing it this way takes some efficiency away. This was a solution so that the existing value would show up on the front end, but I just now realized I can just add a std field to the edd_get_registered_settings function 🤦‍♂ 

I've made the adjustment.